src/message/ssdp.rs

```diff
@@ -54,18 +57,23 @@ impl SSDPMessage {
     ///
     /// The host header field will be taken care of by the underlying library.
     pub fn send<A: ToSocketAddrs, C, S>(&self, connector: &mut C, dst_addr: A) -> SSDPResult<()>
```

Comment:
Derp... good catch. I'll filter the result of the connectors based on the type of address passed to the unicast.